docs/instructions.md

```diff
@@ -0,0 +1,69 @@
+# Metal³ user-guide instructions
+
+We are using [Mdbook](https://github.com/rust-lang/mdBook) to build the
+Metal³ user-guide. Below you will find step-by-step inctructions on how
+to test your changes.
+
+## User-guide structure
+
+Below is the concatenated file structure for the Metal³ user-guide.
+
+```shell
+├── book.toml
+├── src
+│   ├── images
+│   │   └── metal3-color.svg
+│   ├── introduction.md
+│   ├── project-overview.md
+│   └── SUMMARY.md
+└── theme
+    └── favicon.svg
+```
+
+### src
+
+`src` folder stores two main files consumed by the Mdbook,
+apart from the actual content/files. These are book.toml and SUMMARY.md
+
+### book.toml
+
+All parameters and configurations of the user-guide is done via book.toml.
+These include output parameters, redirects, metadata such as title,
+description, authors, language, etc. More information on that can be found
+[here](https://rust-lang.github.io/mdBook/format/config.html).
+
+### SUMMARY.md
+
+This is a context of the user-guide and defines the exact structure.
+Based on any order of documents given in the STRUCTURE.md, mdbook will
+try to fetch those documents and parse them out.
+
+## Preview your changes locally
+
+1. Download Mdbook binary. This will save the binary in /hack folder.
+
+    ```bash
+    $ make bin
+    ```
+
+1. Build the user-guide.
```

Comment:
```suggestion
2. Build the user-guide.
```

Reply:
This isn't a typo. It is a functionality of markdown, to give ordered number based on that. See the screenshot
![docs](https://user-images.githubusercontent.com/35802557/103515502-825ade00-4e77-11eb-9eca-9dcc3f136026.png)

Reply:
Was not aware of that, thanks!